lib/danger/helpers/comments_helper.rb

```diff
@@ -46,6 +46,8 @@ def messages_are_equivalent(m1, m2)
         m1 == m2
       end
 
+      # @endgroup
```

Comment:
Without `@endgroup`, the rest lines in this file are treated as in the same group.